### PR TITLE
Upgrade cypress

### PR DESCRIPTION
**Pull request by nh758**

### CODE REVIEW

File: lib/tasks/configTenantAdmin.js [hunk 1]
```diff
@@ -47,8 +47,9 @@ Command.run = function (options) {
                done();
             },
             questions,
+            writeEnvironment,
             updateSQLInsert,
-            updateTestConfig,
+            // updateTestConfig,
```
Comment (Hiro-Nakamura) [inline lib/tasks/configTenantAdmin.js]: so, do we not need this fn() any more?  See CodeQL warning below.

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]
other than what to do with the unused fn(), looks good to me.

File: lib/tasks/configTenantAdmin.js (hunk 1 above)
Comment (nh758) [reply lib/tasks/configTenantAdmin.js]: no we don't, it patches files that no longer exist. removed now

**Review by jinzsom — APPROVED** [review]
LGTM

**Pull request merged**